### ISSUE TITLE
Version: 0.10.12 - P4 Plugin Error: Internal error: null when performing different actions

### ISSUE THREAD

**Issue by quicktime** [groboclown/p4ic4idea#210]
**Describe the bug**
Constantly getting P4 Plugin Error: Internal error: null when performing different actions.

**To Reproduce**
Many ways to reproduce, almost all P4 actions produce this error.
Setup connection (under Configuration Problems, 0 errors, 1 warnings: "Unknown error"
I am able to connect and have an active connection to my server/workspace.

Refresh Local Changes
Error appears

**Expected behavior**
Error should not appear

**Attachments**
```
2020-02-26 14:18:44,973 [ 250308]  DEBUG -                         p4.api - p4java info: P4Java server factory called for Perforce server URI: p4jrpcnts://REDACTED_PORT_NAME 
2020-02-26 14:18:44,973 [ 250308]  DEBUG -                         p4.api - p4java info: Using Server implementation class: com.perforce.p4java.impl.mapbased.rpc.NtsServerImpl 
2020-02-26 14:18:44,973 [ 250308]  DEBUG -                         p4.api - p4java info: Using program name: 'intellij-perforce-community-plugin-connection'; program version: '0.10.12' 
2020-02-26 14:18:44,973 [ 250308]  DEBUG -                         p4.api - p4java info: Using tmp file directory: REDACTED_FILE_NAME 
2020-02-26 14:18:44,974 [ 250309]  DEBUG -                         p4.api - p4java info: RPC connection to Perforce server REDACTED_PORT_NAME established 
2020-02-26 14:18:44,974 [ 250309]  DEBUG -                         p4.api - p4java info: connected to Perforce server at REDACTED_PORT_NAME 
2020-02-26 14:18:44,974 [ 250309]   INFO -                             p4 - cmd (1): p4 info 
2020-02-26 14:18:44,984 [ 250319]  DEBUG -                             p4 - Command 1 ran in 10 milliseconds 
2020-02-26 14:18:44,984 [ 250319]   INFO -                             p4 - cmd (2): p4 client -o REDACTED_CLIENT_NAME 
2020-02-26 14:18:44,990 [ 250325]  DEBUG -                             p4 - Command 2 ran in 6 milliseconds 
2020-02-26 14:18:44,991 [ 250326]   INFO - onnection.ConnectCommandRunner - Starting listOpenedFilesChanges at Wed Feb 26 14:18:44 PST 2020 
2020-02-26 14:18:44,991 [ 250326]   INFO -                             p4 - cmd (3): p4 changes -i -cREDACTED_CLIENT_NAME -m500 -spending -uREDACTED_USERNAME 
2020-02-26 14:18:44,996 [ 250331]  DEBUG -                             p4 - Command 3 ran in 5 milliseconds 
2020-02-26 14:18:44,996 [ 250331]   INFO -                             p4 - cmd (4): p4 change -s -o 217905 
2020-02-26 14:18:45,002 [ 250337]  DEBUG -                             p4 - Command 4 ran in 6 milliseconds 
2020-02-26 14:18:45,002 [ 250337]   INFO -                             p4 - cmd (5): p4 describe -s 217905 
2020-02-26 14:18:45,004 [ 250339]  DEBUG -                             p4 - Command 5 ran in 2 milliseconds 
2020-02-26 14:18:45,005 [ 250340]   INFO -                             p4 - cmd (6): p4 fstat -m5000 -edefault -Rc -Ro -Ol -Op -Oh //REDACTED_CLIENT_NAME/... 
2020-02-26 14:18:45,010 [ 250345]  DEBUG -                             p4 - Command 6 ran in 5 milliseconds 
2020-02-26 14:18:45,010 [ 250345]   INFO -                             p4 - cmd (7): p4 where REDACTED_FILE_NAME REDACTED_FILE_NAME REDACTED_FILE_NAME
2020-02-26 14:18:45,013 [ 250348]  DEBUG -                             p4 - Command 7 ran in 3 milliseconds 
2020-02-26 14:18:45,013 [ 250348]   INFO - onnection.ConnectCommandRunner - Finished listOpenedFilesChanges; started at Wed Feb 26 14:18:44 PST 2020, ended at Wed Feb 26 14:18:45 PST 2020 
2020-02-26 14:18:45,013 [ 250348]  DEBUG -                         p4.api - p4java info: Disconnected RPC connection to Perforce server REDACTED_PORT_NAME 
2020-02-26 14:18:45,014 [ 250349]  DEBUG -                         p4.api - p4java info: disconnected from Perforce server at REDACTED_PORT_NAME 
2020-02-26 14:18:45,014 [ 250349]  DEBUG -                       p4.stats - p4java stats: RPC connection connected for 40 msec elapsed time 
2020-02-26 14:18:45,014 [ 250349]  DEBUG -                       p4.stats - p4java stats: RPC server connections made: 1 
2020-02-26 14:18:45,014 [ 250349]  DEBUG -                       p4.stats - p4java stats: RPC send ByteBuffer size: 0 
2020-02-26 14:18:45,014 [ 250349]  DEBUG -                       p4.stats - p4java stats: RPC receive ByteBuffer size: 0 
2020-02-26 14:18:45,014 [ 250349]  DEBUG -                       p4.stats - p4java stats: RPC packets sent: 12; RPC packets received: 19 
2020-02-26 14:18:45,014 [ 250349]  DEBUG -                       p4.stats - p4java stats: stream sends: 12; stream recvs: 38 
2020-02-26 14:18:45,014 [ 250349]  DEBUG -                       p4.stats - p4java stats: bytes sent: 3364; bytes received: 5379 bytes 
2020-02-26 14:18:45,014 [ 250349]  DEBUG -                       p4.stats - p4java stats: largest RPC packet sent (bytes): 655; largest RPC packet receieved (bytes): 882 
2020-02-26 14:18:45,014 [ 250349]  DEBUG -                       p4.stats - p4java stats: largest socket send (bytes): 655; largest socket recv (bytes): 882 
2020-02-26 14:18:45,014 [ 250349]  DEBUG -                       p4.stats - p4java stats: RPC put buffer resizes: 0 
2020-02-26 14:18:45,014 [ 250349]  DEBUG -                       p4.stats - p4java stats: RPC read buffer incomplete reads: 0 
2020-02-26 14:18:45,014 [ 250349]   WARN - nnection.P4RequestErrorHandler - Running an action with the Perforce server client:REDACTED_PORT_NAME@REDACTED_CLIENT_NAME generated an error. 
java.lang.UnsupportedOperationException
	at java.base/java.util.AbstractList.add(AbstractList.java:153)
	at java.base/java.util.AbstractList.add(AbstractList.java:111)
	at java.base/java.util.AbstractCollection.addAll(AbstractCollection.java:352)
	at net.groboclown.p4.server.impl.connection.ConnectCommandRunner.listOpenedFilesChanges(ConnectCommandRunner.java:1035)
	at net.groboclown.p4.server.impl.connection.ConnectCommandRunner.lambda$listOpenedFilesChanges$29(ConnectCommandRunner.java:338)
	at net.groboclown.p4.server.impl.connection.impl.SimpleConnectionManager.lambda$null$0(SimpleConnectionManager.java:124)
	at net.groboclown.p4.server.impl.connection.P4RequestErrorHandler.handleConnection(P4RequestErrorHandler.java:156)
	at net.groboclown.p4.server.impl.connection.P4RequestErrorHandler.handle(P4RequestErrorHandler.java:54)
	at net.groboclown.p4.server.impl.connection.impl.SimpleConnectionManager.lambda$handleAsync$5(SimpleConnectionManager.java:578)
	at net.groboclown.p4.server.impl.connection.impl.SimpleConnectionManager.lambda$startPromise$8(SimpleConnectionManager.java:596)
	at net.groboclown.p4.server.api.async.AsyncAnswer.lambda$background$0(AsyncAnswer.java:49)
	at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:238)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-02-26 14:18:45,014 [ 250349]   WARN - .components.UserErrorComponent -  
java.lang.UnsupportedOperationException
	at java.base/java.util.AbstractList.add(AbstractList.java:153)
	at java.base/java.util.AbstractList.add(AbstractList.java:111)
	at java.base/java.util.AbstractCollection.addAll(AbstractCollection.java:352)
	at net.groboclown.p4.server.impl.connection.ConnectCommandRunner.listOpenedFilesChanges(ConnectCommandRunner.java:1035)
	at net.groboclown.p4.server.impl.connection.ConnectCommandRunner.lambda$listOpenedFilesChanges$29(ConnectCommandRunner.java:338)
	at net.groboclown.p4.server.impl.connection.impl.SimpleConnectionManager.lambda$null$0(SimpleConnectionManager.java:124)
	at net.groboclown.p4.server.impl.connection.P4RequestErrorHandler.handleConnection(P4RequestErrorHandler.java:156)
	at net.groboclown.p4.server.impl.connection.P4RequestErrorHandler.handle(P4RequestErrorHandler.java:54)
	at net.groboclown.p4.server.impl.connection.impl.SimpleConnectionManager.lambda$handleAsync$5(SimpleConnectionManager.java:578)
	at net.groboclown.p4.server.impl.connection.impl.SimpleConnectionManager.lambda$startPromise$8(SimpleConnectionManager.java:596)
	at net.groboclown.p4.server.api.async.AsyncAnswer.lambda$background$0(AsyncAnswer.java:49)
	at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:238)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-02-26 14:18:45,014 [ 250349]   INFO - n.impl.SimpleConnectionManager - Command execution failed 
net.groboclown.p4.server.api.P4CommandRunner$ServerResultException: Unknown error
	at net.groboclown.p4.server.impl.connection.P4RequestErrorHandler.createServerResultException(P4RequestErrorHandler.java:131)
	at net.groboclown.p4.server.impl.connection.impl.MessageP4RequestErrorHandler.handleException(MessageP4RequestErrorHandler.java:511)
	at net.groboclown.p4.server.impl.connection.P4RequestErrorHandler.handleConnection(P4RequestErrorHandler.java:161)
	at net.groboclown.p4.server.impl.connection.P4RequestErrorHandler.handle(P4RequestErrorHandler.java:54)
	at net.groboclown.p4.server.impl.connection.impl.SimpleConnectionManager.lambda$handleAsync$5(SimpleConnectionManager.java:578)
	at net.groboclown.p4.server.impl.connection.impl.SimpleConnectionManager.lambda$startPromise$8(SimpleConnectionManager.java:596)
	at net.groboclown.p4.server.api.async.AsyncAnswer.lambda$background$0(AsyncAnswer.java:49)
	at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:238)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.UnsupportedOperationException
	at java.base/java.util.AbstractList.add(AbstractList.java:153)
	at java.base/java.util.AbstractList.add(AbstractList.java:111)
	at java.base/java.util.AbstractCollection.addAll(AbstractCollection.java:352)
	at net.groboclown.p4.server.impl.connection.ConnectCommandRunner.listOpenedFilesChanges(ConnectCommandRunner.java:1035)
	at net.groboclown.p4.server.impl.connection.ConnectCommandRunner.lambda$listOpenedFilesChanges$29(ConnectCommandRunner.java:338)
	at net.groboclown.p4.server.impl.connection.impl.SimpleConnectionManager.lambda$null$0(SimpleConnectionManager.java:124)
	at net.groboclown.p4.server.impl.connection.P4RequestErrorHandler.handleConnection(P4RequestErrorHandler.java:156)
	... 10 more
2020-02-26 14:18:45,015 [ 250350]   INFO - gin.extension.P4ChangeProvider - Processing changes in REDACTED_FILE_NAME
2020-02-26 14:18:49,088 [ 254423]  DEBUG -                         p4.api - p4java info: P4Java server factory called for Perforce server URI: p4jrpcnts://REDACTED_PORT_NAME 
2020-02-26 14:18:49,088 [ 254423]  DEBUG -                         p4.api - p4java info: Using Server implementation class: com.perforce.p4java.impl.mapbased.rpc.NtsServerImpl 
2020-02-26 14:18:49,088 [ 254423]  DEBUG -                         p4.api - p4java info: Using program name: 'intellij-perforce-community-plugin-connection'; program version: '0.10.12' 
2020-02-26 14:18:49,088 [ 254423]  DEBUG -                         p4.api - p4java info: Using tmp file directory: REDACTED_FILE_NAME
2020-02-26 14:18:49,090 [ 254425]  DEBUG -                         p4.api - p4java info: RPC connection to Perforce server REDACTED_PORT_NAME established 
2020-02-26 14:18:49,090 [ 254425]  DEBUG -                         p4.api - p4java info: connected to Perforce server at REDACTED_PORT_NAME 
2020-02-26 14:18:49,090 [ 254425]   INFO -                             p4 - cmd (1): p4 info 
2020-02-26 14:18:49,100 [ 254435]  DEBUG -                             p4 - Command 1 ran in 10 milliseconds 
2020-02-26 14:18:49,101 [ 254436]   INFO -                             p4 - cmd (2): p4 client -o REDACTED_CLIENT_NAME 
2020-02-26 14:18:49,110 [ 254445]  DEBUG -                             p4 - Command 2 ran in 10 milliseconds 
2020-02-26 14:18:49,110 [ 254445]   INFO - onnection.ConnectCommandRunner - Starting listOpenedFilesChanges at Wed Feb 26 14:18:49 PST 2020 
2020-02-26 14:18:49,110 [ 254445]   INFO -                             p4 - cmd (3): p4 changes -i -cREDACTED_CLIENT_NAME -m500 -spending -uREDACTED_USERNAME 
2020-02-26 14:18:49,116 [ 254451]  DEBUG -                             p4 - Command 3 ran in 6 milliseconds 
2020-02-26 14:18:49,116 [ 254451]   INFO -                             p4 - cmd (4): p4 change -s -o 217905 
2020-02-26 14:18:49,122 [ 254457]  DEBUG -                             p4 - Command 4 ran in 6 milliseconds 
2020-02-26 14:18:49,122 [ 254457]   INFO -                             p4 - cmd (5): p4 describe -s 217905 
2020-02-26 14:18:49,126 [ 254461]  DEBUG -                             p4 - Command 5 ran in 4 milliseconds 
2020-02-26 14:18:49,126 [ 254461]   INFO -                             p4 - cmd (6): p4 fstat -m5000 -edefault -Rc -Ro -Ol -Op -Oh //REDACTED_CLIENT_NAME/... 
2020-02-26 14:18:49,133 [ 254468]  DEBUG -                             p4 - Command 6 ran in 7 milliseconds 
2020-02-26 14:18:49,133 [ 254468]   INFO -                             p4 - cmd (7): p4 where REDACTED_FILE_NAME REDACTED_FILE_NAME REDACTED_FILE_NAME
2020-02-26 14:18:49,136 [ 254471]  DEBUG -                             p4 - Command 7 ran in 3 milliseconds 
2020-02-26 14:18:49,136 [ 254471]   INFO - onnection.ConnectCommandRunner - Finished listOpenedFilesChanges; started at Wed Feb 26 14:18:49 PST 2020, ended at Wed Feb 26 14:18:49 PST 2020 
2020-02-26 14:18:49,136 [ 254471]  DEBUG -                         p4.api - p4java info: Disconnected RPC connection to Perforce server REDACTED_PORT_NAME 
2020-02-26 14:18:49,136 [ 254471]  DEBUG -                         p4.api - p4java info: disconnected from Perforce server at REDACTED_PORT_NAME 
2020-02-26 14:18:49,136 [ 254471]  DEBUG -                       p4.stats - p4java stats: RPC connection connected for 46 msec elapsed time 
2020-02-26 14:18:49,136 [ 254471]  DEBUG -                       p4.stats - p4java stats: RPC server connections made: 1 
2020-02-26 14:18:49,136 [ 254471]  DEBUG -                       p4.stats - p4java stats: RPC send ByteBuffer size: 0 
2020-02-26 14:18:49,136 [ 254471]  DEBUG -                       p4.stats - p4java stats: RPC receive ByteBuffer size: 0 
2020-02-26 14:18:49,136 [ 254471]  DEBUG -                       p4.stats - p4java stats: RPC packets sent: 12; RPC packets received: 19 
2020-02-26 14:18:49,136 [ 254471]  DEBUG -                       p4.stats - p4java stats: stream sends: 12; stream recvs: 38 
2020-02-26 14:18:49,136 [ 254471]  DEBUG -                       p4.stats - p4java stats: bytes sent: 3364; bytes received: 5379 bytes 
2020-02-26 14:18:49,136 [ 254471]  DEBUG -                       p4.stats - p4java stats: largest RPC packet sent (bytes): 655; largest RPC packet receieved (bytes): 882 
2020-02-26 14:18:49,136 [ 254471]  DEBUG -                       p4.stats - p4java stats: largest socket send (bytes): 655; largest socket recv (bytes): 882 
2020-02-26 14:18:49,136 [ 254471]  DEBUG -                       p4.stats - p4java stats: RPC put buffer resizes: 0 
2020-02-26 14:18:49,136 [ 254471]  DEBUG -                       p4.stats - p4java stats: RPC read buffer incomplete reads: 0 
2020-02-26 14:18:49,136 [ 254471]   WARN - nnection.P4RequestErrorHandler - Running an action with the Perforce server client:REDACTED_PORT_NAME@REDACTED_CLIENT_NAME generated an error. 
java.lang.UnsupportedOperationException
	at java.base/java.util.AbstractList.add(AbstractList.java:153)
	at java.base/java.util.AbstractList.add(AbstractList.java:111)
	at java.base/java.util.AbstractCollection.addAll(AbstractCollection.java:352)
	at net.groboclown.p4.server.impl.connection.ConnectCommandRunner.listOpenedFilesChanges(ConnectCommandRunner.java:1035)
	at net.groboclown.p4.server.impl.connection.ConnectCommandRunner.lambda$listOpenedFilesChanges$29(ConnectCommandRunner.java:338)
	at net.groboclown.p4.server.impl.connection.impl.SimpleConnectionManager.lambda$null$0(SimpleConnectionManager.java:124)
	at net.groboclown.p4.server.impl.connection.P4RequestErrorHandler.handleConnection(P4RequestErrorHandler.java:156)
	at net.groboclown.p4.server.impl.connection.P4RequestErrorHandler.handle(P4RequestErrorHandler.java:54)
	at net.groboclown.p4.server.impl.connection.impl.SimpleConnectionManager.lambda$handleAsync$5(SimpleConnectionManager.java:578)
	at net.groboclown.p4.server.impl.connection.impl.SimpleConnectionManager.lambda$startPromise$8(SimpleConnectionManager.java:596)
	at net.groboclown.p4.server.api.async.AsyncAnswer.lambda$background$0(AsyncAnswer.java:49)
	at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:238)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-02-26 14:18:49,137 [ 254472]   WARN - .components.UserErrorComponent -  
java.lang.UnsupportedOperationException
	at java.base/java.util.AbstractList.add(AbstractList.java:153)
	at java.base/java.util.AbstractList.add(AbstractList.java:111)
	at java.base/java.util.AbstractCollection.addAll(AbstractCollection.java:352)
	at net.groboclown.p4.server.impl.connection.ConnectCommandRunner.listOpenedFilesChanges(ConnectCommandRunner.java:1035)
	at net.groboclown.p4.server.impl.connection.ConnectCommandRunner.lambda$listOpenedFilesChanges$29(ConnectCommandRunner.java:338)
	at net.groboclown.p4.server.impl.connection.impl.SimpleConnectionManager.lambda$null$0(SimpleConnectionManager.java:124)
	at net.groboclown.p4.server.impl.connection.P4RequestErrorHandler.handleConnection(P4RequestErrorHandler.java:156)
	at net.groboclown.p4.server.impl.connection.P4RequestErrorHandler.handle(P4RequestErrorHandler.java:54)
	at net.groboclown.p4.server.impl.connection.impl.SimpleConnectionManager.lambda$handleAsync$5(SimpleConnectionManager.java:578)
	at net.groboclown.p4.server.impl.connection.impl.SimpleConnectionManager.lambda$startPromise$8(SimpleConnectionManager.java:596)
	at net.groboclown.p4.server.api.async.AsyncAnswer.lambda$background$0(AsyncAnswer.java:49)
	at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:238)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-02-26 14:18:49,137 [ 254472]   INFO - n.impl.SimpleConnectionManager - Command execution failed 
net.groboclown.p4.server.api.P4CommandRunner$ServerResultException: Unknown error
	at net.groboclown.p4.server.impl.connection.P4RequestErrorHandler.createServerResultException(P4RequestErrorHandler.java:131)
	at net.groboclown.p4.server.impl.connection.impl.MessageP4RequestErrorHandler.handleException(MessageP4RequestErrorHandler.java:511)
	at net.groboclown.p4.server.impl.connection.P4RequestErrorHandler.handleConnection(P4RequestErrorHandler.java:161)
	at net.groboclown.p4.server.impl.connection.P4RequestErrorHandler.handle(P4RequestErrorHandler.java:54)
	at net.groboclown.p4.server.impl.connection.impl.SimpleConnectionManager.lambda$handleAsync$5(SimpleConnectionManager.java:578)
	at net.groboclown.p4.server.impl.connection.impl.SimpleConnectionManager.lambda$startPromise$8(SimpleConnectionManager.java:596)
	at net.groboclown.p4.server.api.async.AsyncAnswer.lambda$background$0(AsyncAnswer.java:49)
	at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:238)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.UnsupportedOperationException
	at java.base/java.util.AbstractList.add(AbstractList.java:153)
	at java.base/java.util.AbstractList.add(AbstractList.java:111)
	at java.base/java.util.AbstractCollection.addAll(AbstractCollection.java:352)
	at net.groboclown.p4.server.impl.connection.ConnectCommandRunner.listOpenedFilesChanges(ConnectCommandRunner.java:1035)
	at net.groboclown.p4.server.impl.connection.ConnectCommandRunner.lambda$listOpenedFilesChanges$29(ConnectCommandRunner.java:338)
	at net.groboclown.p4.server.impl.connection.impl.SimpleConnectionManager.lambda$null$0(SimpleConnectionManager.java:124)
	at net.groboclown.p4.server.impl.connection.P4RequestErrorHandler.handleConnection(P4RequestErrorHandler.java:156)
	... 10 more
2020-02-26 14:18:49,138 [ 254473]   INFO - gin.extension.P4ChangeProvider - Processing changes for [REDACTED_FILE_NAME] under REDACTED_FILE_NAME
```

**Desktop (please complete the following information):**
IntelliJ IDEA 2019.3.3 (Community Edition)
Build #IC-193.6494.35, built on February 10, 2020
Runtime version: 11.0.5+10-b520.38 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Windows 10 10.0

**Comment by groboclown** [comment]
Thanks for the quick check on this.  Looks like this happens particularly when no files are found on a query.  I'll get a patch out quick.

**Comment by quicktime** [comment]
Fixed in build 0.10.14

**Comment by groboclown** [comment]
Thanks for verifying!
